policy/modules/services/ppp.te

```diff
@@ -211,6 +211,10 @@ optional_policy(`
 	postfix_domtrans_master(pppd_t)
 ')
 
+optional_policy(`
+    resolvconf_domtrans(pppd_t)
```

Comment:
Please review the whitespace, there should be tabs instead of spaces. Please recheck the entire changeset.